src/stars_star.css

```diff
@@ -0,0 +1,7 @@
+.stars_star {
+  background-image: url(images/star-grey.png);
+  width: 16px;
+  height: 16px;
+  display: flex;
+  margin-right: 20px;
```

Comment:
This margin is wrong, Figma shows a margin of 4px between stars.

Also, you should prevent the last star from having that margin, using `stars__star:last-child`